string write_data
---
bool status_of_write_data
